meson_options.txt

```diff
@@ -153,8 +153,8 @@ option(
 
 option(
     'with_libsodium',
-    type : 'boolean',
-    value : 'true',
+    type : 'feature',
+    value : 'auto',
     description : 'build nix with shared libs')
```

Comment:
was this description always wrong? :O

Reply:
Oh haha yeah it was.

Reply:
@p01arst0rm ah did you mean for me to fix this too, now that we are noticing it?

Reply:
yep ^^